### ISSUE TITLE
$debugName doesn't work with keyframes

### ISSUE THREAD

**Issue by jamiewinder** [typestyle/typestyle#85]
Not sure if it *should* work or not, but the equivalent feature in free style - display name - works for both classes and keyframes.

Example: http://typestyle.io/play/#src=const%20test%20=%20keyframes(%7B%0A%20%20$debugName:%20'test',%0A%20%20from:%20%7B%20color:%20'red'%20%7D,%0A%20%20to:%20%7B%20color:%20'blue'%20%7D%0A%7D);%0A

```
EVAL ERROR: Cannot assign to read only property '0' of string 'test'
```

**Comment by notoriousb1t** [comment]
Looking at this now

**Comment by notoriousb1t** [comment]
Fixed in https://github.com/typestyle/typestyle/commit/158cad9f7491a34d0a2224c70d38f51d15daa09b, Released in v0.22.1 😄
